packages/sdk/Dockerfile

```diff
@@ -169,5 +169,67 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+# Install migrate db cli
+ARG GO_MIGRATE_VERSION=4.18.2
+RUN <<EOF
+ARCH=$(dpkg --print-architecture || uname -m)
+case "$ARCH" in
+    amd64|x86_64)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-amd64.deb"
+        ;;
+    arm64|aarch64)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-arm64.deb"
+        ;;
+    armhf|armv7l)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-armv7.deb"
+        ;;
+    i386)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-386.deb"
+        ;;
+    *)
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+        ;;
+esac
+
+curl -fsSL "$MIGRATE_URL" -o /tmp/migrate.deb
+dpkg -i /tmp/migrate.deb
+rm /tmp/migrate.deb
+migrate --version
+EOF
+
+
+# Get migration files for rollups-graphql
+ARG GRAPHQL_REPO=rollups-graphql
+ARG CARTESI_ROLLUPS_GRAPHQL_VERSION=2.3.8
+ARG GRAPHQL_GIT_FOLDER=dev/migrations
+ARG GRAPHQL_DEST_FOLDER=db/rollups-graphql
```

Comment:
Other than `CARTESI_ROLLUPS_GRAPHQL_VERSION`, I don't see value in making those other build variables.

They could be hardcoded or  environment variables  inside the `RUN <<EOF` scope.

Reply:
I agree, changed